ast/options.go

```diff
@@ -23,11 +23,11 @@ func (o *Options) Field(name string) (expr Expr, found bool) {
 
 // FieldNotFound is returned Options.*Field methods.
 // It can be handled as a non-error.
-var FieldNotFound = errors.New("field not found")
+var ErrFieldNotFound = errors.New("field not found")
 
 // fieldTypeMismatch is only defined for test.
 // It is intentionally unexported.
-var fieldTypeMismatch = errors.New("type mismatched")
+var errFieldTypeMismatch = errors.New("type mismatched")
```

Comment:
@apstndb A question unrelated to this issue: Why is this value not exported?

Reply:
It is because user should only check `ErrFieldNotFound` which is not error state. Other errors will be handled as unrecoverable error.
If user want to handle unexpected types, `(*Options).Field(name string) (Expr, bool)` should be used.